### PR TITLE
i18n - ran `make -C locale tx-update`

### PR DESCRIPTION
**Pull request by ianballou**

### CODE REVIEW

File: locale/foreman.pot [hunk 1]
```diff
@@ -16714,6 +16685,314 @@ msgstr ""
 msgid "Connected to: %s"
 msgstr ""
 
+#: action_names.rb:2
```
Comment (ekohl) [inline locale/foreman.pot]: I think you ran extraction with plugins (tasks / Katello?) enabled, so it also included all those tasks. Sadly foreman-tasks can't separate them so you should run extraction without plugins.

**Review by ekohl — CHANGES_REQUESTED** [review]

File: locale/foreman.pot (hunk 1 above)
Comment (ianballou) [reply locale/foreman.pot]: I don't have a Foreman-only dev env up... if anyone beats me to it, feel free to make a separate PR.

Comment (ekohl) [reply locale/foreman.pot]: I think it's sufficient to disable all gems in the short term, then re-enable after it.

Comment (ianballou) [reply locale/foreman.pot]: I'm not sure why, but disabling the gems + the katello config file is still resulting in Katello stuff making it into the `.pot` file.

**Comment by ianballou** [comment]
@ekohl I re-ran the string extraction with a vanilla foreman development environment.

**Review by ekohl — APPROVED** [review]

**Comment by ekohl** [comment]
Will wait for CI to complete, but :+1:

**Pull request merged**